code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1283,22 +1281,163 @@
-	health_required = 40
+	description = "Hello! I am a bugged reagent. Please report me for my crimes. Thank you!!"
+	toxpwr = 0
 	liver_damage_multiplier = 0
+	chemical_flags = REAGENT_NO_RANDOM_RECIPE
 
-/datum/reagent/toxin/viperspider
-	name = "Viper Spider Toxin"
-	toxpwr = 5
-	description = "An extremely toxic chemical produced by the rare viper spider. Brings their prey to the brink of death and causes hallucinations."
-	health_required = 10
-	liver_damage_multiplier = 0
+/datum/reagent/toxin/spider/paralysis
+	name = "Paralysis Spider Toxin"
+	description = "A toxic chemical produced by spiders to incapacitate prey."
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	toxpwr = 0.1
+	liver_damage_multiplier = 1
+	var/list/parlyzed_limbs = list()
+	var/list/unaffected_limbs = list(
+		TRAIT_PARALYSIS_L_ARM = "left arm",
+		TRAIT_PARALYSIS_R_ARM = "right arm",
+		TRAIT_PARALYSIS_L_LEG = "left leg",
+		TRAIT_PARALYSIS_R_LEG = "right leg",
+	)
+
+/datum/reagent/toxin/spider/paralysis/on_mob_life(mob/living/carbon/affected_mob, seconds_per_tick, times_fired)
+	. = ..()
+
+	// every 5u of toxin can paralysis one limb so to get all four you need more than 20u
+	var/max_toxin_limit = 5 * (length(parlyzed_limbs) + 1)
+	var/min_toxin_limit = 5 * (length(parlyzed_limbs))
+
+	if(SPT_PROB(0.5, seconds_per_tick))
+		affected_mob.adjust_slurring(3 SECONDS * REM * seconds_per_tick)
+
+	if(volume <= min_toxin_limit)
+		if(SPT_PROB((min_toxin_limit - volume), seconds_per_tick))
+			unparalyze_limb(affected_mob)
+	else if(volume >= max_toxin_limit)
+		if(SPT_PROB((volume - max_toxin_limit), seconds_per_tick))
+			paralyze_limb(affected_mob)
+
+/datum/reagent/toxin/spider/paralysis/proc/paralyze_limb(mob/living/affected_mob)
+	if(!length(unaffected_limbs))
+		return
+	var/parlyze_limb_trait = pick(unaffected_limbs)
+	var/limb_name = unaffected_limbs[parlyze_limb_trait]
+	ADD_TRAIT(affected_mob, parlyze_limb_trait, type)
+	unaffected_limbs -= parlyze_limb_trait
+	parlyzed_limbs[parlyze_limb_trait] = limb_name
+
+/datum/reagent/toxin/spider/paralysis/proc/unparalyze_limb(mob/living/affected_mob)
+	if(!length(parlyzed_limbs))
+		return
+	var/parlyze_limb_trait = pick(parlyzed_limbs)
+	var/limb_name = parlyzed_limbs[parlyze_limb_trait]
+	REMOVE_TRAIT(affected_mob, parlyze_limb_trait, type)
+	parlyzed_limbs -= parlyze_limb_trait
+	unaffected_limbs[parlyze_limb_trait] = limb_name
+
+/datum/reagent/toxin/spider/paralysis/on_mob_end_metabolize(mob/living/carbon/affected_mob)
+	. = ..()
+	REMOVE_TRAIT(affected_mob, TRAIT_PARALYSIS_L_ARM, type)
+	REMOVE_TRAIT(affected_mob, TRAIT_PARALYSIS_R_ARM, type)
+	REMOVE_TRAIT(affected_mob, TRAIT_PARALYSIS_R_LEG, type)
+	REMOVE_TRAIT(affected_mob, TRAIT_PARALYSIS_L_LEG, type)
+
+/datum/reagent/toxin/spider/exhaustion
+	name = "Exhaustion Spider Toxin"
+	description = "A toxic chemical produced by spiders to exhaust prey while hunting."
+	metabolization_rate = 2 * REAGENTS_METABOLISM
+	data = 15
```

Comment:
Data is a list, and this is not typically how it is managed

```suggestion
	data = list("stamina_damage_per_tick" = 15"
```
Then access it as much
```dm
    data["stamina_damage_per_tick"] = max(data["stamina_damage_per_tick"] - 1, 3)

Reply:
Though ostensibly I don't think this even *needs* to use data, so long as you implement `on_merge`

Reply:
See above. Changed it to a var with f976783557cecb6e7b14ba8c30a53e92e5ed3c4f